src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -127,5 +136,15 @@ private bool UnblockAccount(ApplicationEngine engine, UInt160 account)
             engine.Snapshot.Storages.Delete(key);
             return true;
         }
+
+        [ContractMethod(0_03000000, CallFlags.AllowModifyStates)]
+        private bool SetFeeRatio(ApplicationEngine engine, uint value)
```

Comment:
It must be decimal... and we don't have decimal in vm.

Reply:
we can zoom 10 times.

Reply:
100 times it's good for me